src/extension/background-script/connectors/commando.ts

```diff
@@ -0,0 +1,334 @@
+      ip: this.config.host,
+      port: this.config.port || 9735,
+      privateKey: this.config.privateKey,
+      // logger: {
+      //   info: console.log,
+      //   warn: console.warn,
+      //   error: console.error
+      // },
+    });
+  }
+
+  async init() {
+    // initiate the connection to the remote node
+    await this.ln.connect();
+  }
+
+  async unload() {
+    await this.ln.disconnect();
+  }
+
+  async connectPeer(
+    args: ConnectPeerArgs
+  ): Promise<ConnectPeerResponse | Error> {
+    return this.ln
+      .commando({
+        method: "connect",
+        params: {
+          id: args.pubkey,
+          host: args.host,
+        },
+        rune: this.config.rune,
+      })
+      .then((resp) => {
+        return {
+          data: true,
+        };
+      })
+      .catch((err) => {
+        return new Error(err);
+      });
+  }
+
+  async getInvoices(): Promise<GetInvoicesResponse> {
+    return this.ln
+      .commando({
+        method: "listinvoices",
+        params: {},
+        rune: this.config.rune,
+      })
+      .then((resp) => {
+        const parsed = resp as CommandoListInvoicesResponse;
+        return {
+          data: {
+            invoices: parsed.invoices.map(
+              (invoice, index): ConnectorInvoice => ({
+                id: invoice.label,
+                memo: invoice.description,
+                settled: invoice.status == "paid",
+                preimage: invoice.payment_preimage,
+                settleDate: invoice.paid_at,
+                type: "received",
+                totalAmount: (invoice.amount_received_msat / 1000).toString(),
+              })
+            ),
+          },
+        };
+      });
+  }
+
+  async getInfo(): Promise<GetInfoResponse> {
+    const response = (await this.ln.commando({
+      method: "getinfo",
+      params: {},
+      rune: this.config.rune,
+    })) as CommandoGetInfoResponse;
+    return {
+      data: {
+        alias: response.alias,
+        pubkey: response.id,
+        color: response.color,
+      },
+    };
+  }
+
+  async getBalance(): Promise<GetBalanceResponse> {
+    const response = (await this.ln.commando({
+      method: "listfunds",
+      params: {},
+      rune: this.config.rune,
+    })) as CommandoListFundsResponse;
+    let lnBalance = 0;
+    for (let i = 0; i < response.channels.length; i++) {
+      lnBalance = lnBalance + response.channels[i].channel_sat;
+    }
+    return {
+      data: {
+        balance: lnBalance,
+      },
+    };
+  }
+
+  async sendPayment(args: SendPaymentArgs): Promise<SendPaymentResponse> {
+    return this.ln
+      .commando({
+        method: "pay",
+        params: {
+          bolt11: args.paymentRequest,
+        },
+        rune: this.config.rune,
+      })
+      .then((resp) => {
+        const parsed = resp as CommandoPayInvoiceResponse;
+        const parsedTotalAmtStr = parsed.amount_msat.replace("msat", "");
```

Comment:
puh, don't the return some proper number values without the currency string?

Reply:
Actually they do, I don't know why I missed this. Will update PR

Reply:
It's also just the wrong field, I made several brainfarts here.

Reply:
In this call, they don't (ignore my deleted comments)..
I don't know why actually. Maybe we should ask them about it.

Reply:
strange that a node does not return numbers in an API response. typically you want to calculate with numbers they should not be formatted by default. 
but they have some strange things in their API (like the description hash thing)

Reply:
Turns out the problem was with me